Sources/Clappr_tvOS/Classes/Plugin/Playback/AVFoundationPlayback/AVFoundationPlayback.swift

```diff
@@ -265,13 +265,26 @@ open class AVFoundationPlayback: Playback, AVPlayerViewControllerDelegate {
     open override func seek(_ timeInterval: TimeInterval) {
         let time = CMTimeMakeWithSeconds(timeInterval, Int32(NSEC_PER_SEC))
 
-        player?.currentItem?.seek(to: time)
         trigger(.seek)
+        trigger(.willSeek)
+
+        player?.currentItem?.seek(to: time) { [weak self] success in
+            if success {
+                self?.trigger(.didSeek)
+            }
+        }
+
         trigger(.positionUpdate, userInfo: ["position": CMTimeGetSeconds(time)])
     }
 
+    public func playerViewController(_ playerViewController: AVPlayerViewController, timeToSeekAfterUserNavigatedFrom oldTime: CMTime, to targetTime: CMTime) -> CMTime {
+        trigger(.willSeek)
+        return targetTime
+    }
+
     public func playerViewController(_ playerViewController: AVPlayerViewController, willResumePlaybackAfterUserNavigatedFrom oldTime: CMTime, to targetTime: CMTime) {
         trigger(.seek)
```

Comment:
on Seek method the old seek event was triggered with willseek, here is triggered with didSeek.
I think that is better to remove the old .seek event from the Project.

Reply:
The previous behavior of seek event was act as a willSeek event. In our metrics plugins the most correct behavior is to listen to didSeek.

We are proposing to keep seek, but deprecate it. We need to update the metrics plugins to listen to the correct event in another task. Our plan is to keep it for now and remove it when we update our metrics plugins.

What you think about that?